### PR TITLE
feat: Add a possibility to execute Power Shell scripts

### PR DESCRIPTION
**Pull request by mykola-mokhnach**
Just because we can. I'll also have to write some documentation for it later and build other features that need PS

### CODE REVIEW

File: lib/commands/powershell.js [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+};
+const POWER_SHELL = 'powershell.exe';
+const POWER_SHELL_SCRIPT_PATTERN = /^powerShell$/;
+const POWER_SHELL_FEATURE = 'power_shell';
+
+const commands = {};
+
+commands.execute = async function execute (script, args) {
+  if (POWER_SHELL_SCRIPT_PATTERN.test(script)) {
+    this.ensureFeatureEnabled(POWER_SHELL_FEATURE);
```
Comment (KazuCocoa) [inline lib/commands/powershell.js:22]: 👍

Comment (KazuCocoa) [reply lib/commands/powershell.js:22]: Add this in readme like https://github.com/appium/appium-xcuitest-driver#opt-in-features-with-security-risk ?

**Review by KazuCocoa — APPROVED** [review]

File: lib/commands/powershell.js (hunk 1 above)
Comment (mykola-mokhnach) [reply lib/commands/powershell.js:22]: Kazu, you know how much I like to write documentation %)

Comment (KazuCocoa) [reply lib/commands/powershell.js:22]: haha :P
(I can assist later)

**Pull request merged**